backend/medtagger/api/scans/service_rest.py

```diff
@@ -137,7 +137,11 @@ def post(scan_id: ScanID) -> Any:
         business.validate_label_payload(elements, files)
 
         labeling_time = label['labeling_time']
-        label = business.add_label(scan_id, elements, files, labeling_time)
+        if 'comment' in label:
```

Comment:
I don't like this if-statement. It would be better to do it like this:

```python
comment = label.get('comment')  # Won't raise an error and return None gracefully
label = business.add_label(scan_id, elements, files, labeling_time, comment)
```